src/libcore/slice/mod.rs

```diff
@@ -650,6 +653,16 @@ impl<T> SliceExt for [T] {
         }
     }
 
+    fn rotate_right(&mut self, mid: usize) {
```

Comment:
nit: `k` here too.

Reply:
good call! addressed in the latest force push